### DISCUSSION TITLE
[Feature]: Financial sponsoring via Github

### DISCUSSION THREAD

**Discussion by YanPes** [web-infra-dev/rspack#8691] [Ideas]
### What problem does this feature solve?

Currently, it is only possible to support rspack financially with [opencollective](https://opencollective.com/rspack)

It would be nice to have to option to also donate money via Github to enable a more transparent, easy to find and standardized access to financial contributions on repositories.
In addition, people get a nice badge as sponsoring partner :-)

### What does the proposed API of configuration look like?

no idea about sponsoring section of repository section.

**Comment by chenjiahan** [comment]
Thank you for your suggestion! We appreciate your interest in supporting Rspack. ❤️

Currently, we plan to continue using Open Collective as our primary platform. We do appreciate the convenience and visibility that GitHub Sponsors provides, including the sponsor badges you mentioned. While we don't have immediate plans to enable GitHub Sponsors, we'll keep this suggestion in mind for future consideration as we evaluate additional funding channels.
